scala/spark-pubsub/cloud-pubsub-receiver/cloudpubsubreceiver.sbt

```diff
@@ -0,0 +1,16 @@
+name := "Cloud Pubsub Receiver"
+
+version := "0.0"
+
+scalaVersion := "2.10.3"
+
+libraryDependencies ++= Seq("org.apache.spark" % "spark-core_2.10" % "1.3.0", 
+      "org.apache.spark" % "spark-streaming_2.10" % "1.3.0",
+      "org.apache.hadoop" % "hadoop-common" % "2.5.2", 
+      "org.apache.hbase" % "hbase-client" % "1.0.1", 
+      "org.apache.hbase" % "hbase-server" % "1.0.1", 
+      "org.apache.hbase" % "hbase-common" % "1.0.1",
+      "org.apache.hbase" % "hbase-annotations" % "1.0.1",
+      "org.mortbay.jetty.alpn" % "alpn-boot" % "7.0.0.v20140317",
```

Comment:
@sduskis @taragu bdutil is wrong then.  alpn-boot needs to match the version of java you are using -- http://www.eclipse.org/jetty/documentation/current/alpn-chapter.html#alpn-versions